incubator/istio/README.md

```diff
@@ -18,7 +18,29 @@ This chart bootstraps a [Istio](https://istio.io/) deployment on a [Kubernetes](
 ## Prerequisites
 
 - Kubernetes 1.5+
-- istioctl - See installation steps [here](https://istio.io/docs/tasks/installing-istio.html#installation-steps)
+- istioctl
+
+### istioctl installation steps
+
+1. Go to the [Istio release](https://github.com/istio/istio/releases) page, to download the installation file corresponding to your OS or run 
+   ```bash
+   curl -L https://git.io/getIstio | sh -
+   ```
+   to download and extract the latest release automatically (on MacOS and Ubuntu).
+
+2. Extract the installation file, and change directory to the location where the files were extracted. The following instructions 
+   are relative to this installation directory.
```

Comment:
I would honestly just link the installation instructions to just in case they change